smac/scenario/scenario.py

```diff
@@ -18,6 +18,14 @@
 __version__ = "0.0.2"
 
 
+def _is_truthy(arg):
+    return arg in ["1", "true", True]
+
+def _parse_initial_incumbent(arg):
+    arg = str.upper(arg.strip())
+    return arg if arg in ['DEFAULT', 'RANDOM'] else None
```

Comment:
If we want to add a "choice" argument (as in argparse) in another parameter, we have to write every time a new function?